great_expectations/rule_based_profiler/estimators/bootstrap_numeric_range_estimator.py

```diff
@@ -79,13 +80,18 @@ def _get_numeric_range_estimate(
             parameters=parameters,
         )
 
+        # Obtain quantile_statistic_interpolation_method override from "rule state" (i.e., variables and parameters); from instance variable otherwise.
```

Comment:
@NathanFarmer Let us not add this comment -- the utility function already does it.

Reply:
✅